ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -607,3 +607,9 @@ def Elu(features, name=None):
         "float64",
     ),
 }
+
+
+# @with_unsupported_dtypes({"2.9.1 and below": ()}, "tensorflow")
+@to_ivy_arrays_and_back
+def LinSpace(*, start, stop, num, name=None):
```

Comment:
These arguments are not keyword-only, you can remove `*`

Reply:
As for this, the test is now failing with an error as follows: 
```sh
E       TypeError: lin_space only takes keyword args (possible keys: ['start', 'stop', 'num', 'name']). Please pass these args as kwargs instead.
../../mambaforge/envs/ivy_dev/lib/python3.8/site-packages/tensorflow/python/util/tf_export.py:396: TypeError
```

Reply:
Here's a [related error](https://stackoverflow.com/questions/65257591/tensorflow2-placeholder-only-takes-keyword-args) I found on the issue, if that helps (apologies if I'm being wrong somewhere).

Reply:
Oh, You're right, I wasn't aware of that. 
Thanks for clearing that up 😄